WORKSPACE

```diff
@@ -34,14 +34,20 @@ bazel_skylib_workspace()
 git_repository(
     name = "com_github_baidu_braft",
     remote = "https://github.com/baidu/braft",
-    commit = "e255c0e4b18d1a8a5d484d4b647f41ff1385ef1e",
+    commit = "351c74aca6f7d5436da0c2e06c93067659c31435",
```

Comment:
it's better to use a stable version rather than a mainline version, and the latest stable is v1.1.2, https://github.com/baidu/braft/releases.

Reply:
I use the stable version v1.0.2. Because the newer version's function `append_entries` need add a parameter.

Reply:
> I use the stable version v1.0.2. Because the newer version's function `append_entries` need add a parameter.

Did you mean `SegmentLogStorage::append_entries` added another parameter `metric`? If so, I think we can just ignore this parameter, so that, we can use braft v1.1.2.

Reply:
Your first version is acceptable,
<img width="813" alt="image" src="https://user-images.githubusercontent.com/10327590/201805506-c4ab3d97-5662-42e3-9ab8-80a7b14a0766.png">